src/fixtures/index.ts

```diff
@@ -0,0 +1,52 @@
+import { test as base } from "@playwright/test";
+import { GaragePage, ProfilePage, WelcomePage } from "../page-objects";
+import { USERS, User } from "../data/dict";
+import { STORAGE_STATE_USER_PATH } from "../data/storage-state";
+
+export type Fixtures = {
+  headerLinks: string[];
+  user: User;
+  userProfilePage: ProfilePage;
+  managerProfilePage: ProfilePage;
+  userGaragePage: GaragePage;
+};
+
+export const test = base.extend<Fixtures>({
+  headerLinks: ["Garage", "Fuel expenses", "Instructions"],
+  user: USERS.lesia,
+  page: async ({ browser }, use) => {
+    const ctx = await browser.newContext({
+      storageState: STORAGE_STATE_USER_PATH,
+    });
+    const page = await ctx.newPage();
+
+    await use(page);
+  },
```

Comment:
I do not recommend overriding `page`  with storage state. Leave some flexibility for future